<launch>
  <node pkg="occ_predictor" type="pred_inference_node" name="pred_infer_node" output="screen" launch-prefix="nice -n 15">
    <param name="service_name" value="/frontier/occ_map/pred"/>
		<param name="model_path" value="$(find occ_predictor)/models/predictor.onnx" />
		<param name="num_threads" value="4"/>

	</node>

</launch>